runtime/tr.source/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -1170,7 +1170,7 @@ static bool generateInlineTest(TR::CodeGenerator * cg, TR::Node * node, TR::Node
       if (cg->needClassAndMethodPointerRelocations())
          unloadableConstInstr[i] = generateRegLitRefInstruction(cg, TR::InstOpCode::getLoadOpCode(), node, scratchReg,(uintptrj_t) guessClassArray[i], TR_ClassPointer, NULL, NULL, NULL);
       else
-         unloadableConstInstr[i] = generateRILInstruction(cg, TR::InstOpCode::LARL, node, scratchReg, (uintptrj_t)guessClassArray[i]);
+         unloadableConstInstr[i] = generateRILInstruction(cg, TR::InstOpCode::LARL, node, scratchReg, guessClassArray[i]);
```

Comment:
Is this supposed to use the `void*` overload? Seems to be loading an address?

Reply:
Yes. It is a `LARL` instruction so it uses an address and the `void*` overload.